tools/merge-docker-images.sh

```diff
@@ -36,6 +37,8 @@ docker manifest annotate "${IMAGE_NAME}":$FULL_TAG \
 
 docker manifest push "${IMAGE_NAME}":$FULL_TAG
 
+# if undefined, use the empty string
+GITHUB_REF="${GITHUB_REF-}"
```

Comment:
we have a version of this script floating around that uses a `[ -n "${GITHUB_REF" ]` check. I'm not sure when that comes up, but just in case we do have undefined `GITHUB_REF` I wanted to do this so that the script doesn't fail.